modules/opp-architecture-installing-policyset.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="opp-architecture-installing-policyset_{context}"]
+
+= Installing {product-title} by using policy sets
+
+The {ocp} installation uses two policy sets to install additional products.
+
+[NOTE]
+====
+Edit the `policyGenerator.yaml` file to remove any products that you do not want to install. You can delete the product entries or comment out the lines.
+====
+
+To install the policy sets using gitops, follow the steps in link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/governance/governance#deploying-policies-using-gitops[Deploying policies using gitops] . Use the actual path to the policy set instead of the path in the example path `policygenerator/policy-sets/stable/openshift-plus`.
+
+To install the policy sets by using CLI, use the following procedure:
+
+.Procedure
+. Install the `PolicyGenerator` plug-in by following the instructions in _Installing and using the PolicyGenerator Kustomize plug-in_.
```

Comment:
```suggestion
. Install the `PolicyGenerator` plugin by following the instructions in _Installing and using the PolicyGenerator Kustomize plug-in_.
```

Reply:
I could see an argument for keeping this as plug-in because of the other one to which it refers.  The SSG advises [plugin](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions:~:text=plugin%20(adjective)-,Description,-%3A%20Use%20to%20distinguish) though